R/rextendr_document.R

```diff
@@ -9,13 +9,13 @@
 #' @inheritParams devtools::document
 #' @return No return value, called for side effects.
 #' @export
-document <- function(pkg = ".", quiet = getOption("usethis.quiet", FALSE), roclets = NULL) {
+document <- function(pkg = ".", quiet = FALSE, roclets = NULL) {
```

Comment:
setting default to `FALSE` as to not rely on global usethis option per reasons above.

Reply:
This one isn't a problem imo. It will default to `FALSE` if the option hasn't been set anyway.